testing/base/BUILD

```diff
@@ -104,3 +110,27 @@ cc_test(
         "@googletest//:gtest",
     ],
 )
+
+cc_library(
+    name = "testing_exe_path",
+    testonly = 1,
+    srcs = ["testing_exe_path.cpp"],
```

Comment:
Why isn't testing_exe_path.cpp in this change?

Reply:
Because I forgot to `git add` it. Doh!